stacks/api/function/src/function.controller.ts

```diff
@@ -228,7 +228,9 @@ export class FunctionController {
     }
     operators.push(
       last(),
-      switchMap(() => this.engine.compile(fn).catch(() => {})),
+      tap(() => {
+        this.engine.compile(fn).catch(() => {});
+      }),
```

Comment:
If we will make this change, the dependency install bug named SP-348 will be fixed, isn't it?

Reply:
Yes.